test/sim_circuits.json

```diff
@@ -4,7 +4,7 @@
     "circuit": "OPENQASM 2.0;include \"qelib1.inc\";qreg q[2];x q[0];x q[1];\n"
   },
   {
-    "description": "two_qubit_circuit_with_h_z_controlled_x_swap",
-    "circuit": "OPENQASM 2.0;include \"qelib1.inc\";qreg q[2];cx q[0], q[1];h q[0];z q[0];h q[0];swap q[0], q[1];\n"
+    "description": "two_qubit_circuit_with_h_z_controlled_x",
+    "circuit": "OPENQASM 2.0;include \"qelib1.inc\";qreg q[2];cx q[0], q[1];h q[0];z q[0];h q[0];\n"
```

Comment:
> Seems like an unnecessary change.

I changed this because the hybrid amplitude executor does not seem to work well with swap gates. Should I still change it back?
<img width="1183" alt="image" src="https://github.com/tyi1025/dd_eval/assets/56802240/6be0aa24-f76c-4227-b46c-fe0c25835b5c">

Reply:
Could you create a minimal reproducer of that problem and open an issue in DDSIM for that?
SWAPs shouldn't be an issue.

Reply:
https://github.com/cda-tum/mqt-ddsim/issues/269